.eslintrc

```diff
@@ -0,0 +1,113 @@
+{
+  "env": {
+    "browser": true,
+    "node": true
+  },
+
+  "rules": {
+    // Maybe enable?
+    "one-var": 0,
+    "vars-on-top": 0,
+    "eqeqeq": 0,
+    "yoda": [0, "never"],
+    "no-use-before-define": 0,
+    "no-mixed-requires": [0, false],
+
+    // Discuss?
```

Comment:
Right, I'd like to enable all of these, just didn't want to presume to dictate style